### PR TITLE
Multi-speaker TTS config for AC

### PR DESCRIPTION
**Pull request by a-a-egorovich**

### CODE REVIEW

**Comment by eaidova** [comment]
Jenkins ok to test

**Comment by eaidova** [comment]
@a-a-egorovich when you publish model as intel, you do not need to add model.yml files. these files will be published automatically when model package will be updated.

**Comment by vladimir-dudnik** [comment]
@a-a-egorovich please add model to index file
`14:08:58  text-to-speech-en-multi-speaker-0001 not in index.md file`

**Comment by vladimir-dudnik** [comment]
@a-a-egorovich please add model to index.md file
`14:08:58  text-to-speech-en-multi-speaker-0001 not in index.md file`

File: models/intel/text-to-speech-en-multi-speaker-0001/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+| Source framework                              | PyTorch\* |
+
+## Duration prediction model specification
+
+The text-to-speech-en-multi-0001-duration-prediction model is a ForwardTacotron-based duration predictor for symbols.
+
+| Metric                                        | Value     |
+|-----------------------------------------------|-----------|
+| GFlops                                        | 28.75     |
+| MParams                                       | 26.18    |
```
Comment (eaidova) [inline models/intel/text-to-speech-en-multi-speaker-0001/README.md:23]:
```suggestion
| GFlops                                        | 28.75     |
| MParams                                       | 26.18     |
```

File: models/intel/index.md [hunk 2]
```diff
@@ -284,5 +284,15 @@ Deep Learning models for speech synthesis (mel spectrogram generation and wave f
...
 
+
+Deep Learning models for speech synthesis (mel spectrogram generation and wave form generation).
+
+| Model Name | Complexity (GFLOPs) | Size (Mp) |
+|------------|---------------------|-----------|
+| [text-to-speech-en-multi-speaker-0001](./text-to-speech-en-multi-speaker-0001/README.md) |       |        |
+|   text-to-speech-en-multi-0001-duration-prediction                 | 28.75 | 26.18 |
+|   text-to-speech-en-multi-0001-regression                          | 7.81  | 5.12  |
+|   text-to-speech-en-multi-0001-generation                          | 48.38 | 12.77  |
```
Comment (eaidova) [inline models/intel/index.md:295]:
```suggestion
|   text-to-speech-en-multi-0001-generation                          | 48.38 | 12.77 |
```

**Review by eaidova — APPROVED** [review]

File: models/intel/text-to-speech-en-multi-speaker-0001/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
+# text-to-speech-en-multi-speaker-0001 (composite)
```
Comment (vladimir-dudnik) [inline models/intel/text-to-speech-en-multi-speaker-0001/README.md:1]:
```suggestion
# text-to-speech-en-multi-0001 (composite)
```
suggest to have unified base part of the name for composite models, like we have for all other composite models

**Review by vladimir-dudnik — CHANGES_REQUESTED** [review]

File: models/intel/text-to-speech-en-multi-speaker-0001/README.md (hunk 3 above)
Comment (a-a-egorovich) [reply models/intel/text-to-speech-en-multi-speaker-0001/README.md]: Done

**Review by vladimir-dudnik — APPROVED** [review]

**Pull request merged**